view/frontend/web/js/view/payment/method-renderer/adyen-oneclick-method.js

```diff
@@ -372,36 +344,53 @@ define(
                          * @param type
                          * @param token
                          */
-                        renderThreeDS2Component: function(type, token) {
+                        renderThreeDS2Component: function (type, token) {
                             var self = this;
 
                             var threeDS2Node = document.getElementById('threeDS2ContainerOneClick');
 
                             if (type == "IdentifyShopper") {
-                                fullScreenLoader.startLoader();
                                 self.threeDS2Component = checkout.create('threeDS2DeviceFingerprint', {
-                                        fingerprintToken: token,
-                                        onComplete: function(result) {
+                                    fingerprintToken: token,
+                                    onComplete: function (result) {
+                                        threeds2.processThreeDS2(result.data).done(function (responseJSON) {
+                                            self.validateThreeDS2OrPlaceOrder(responseJSON)
+                                        }).error(function () {
                                             fullScreenLoader.stopLoader();
-                                            self.processThreeDS2(result.data);
-                                        }
-                                    });
+                                        });
+                                    },
+                                    onError: function () {
+                                        fullScreenLoader.stopLoader();
```

Comment:
Would be nice to add the same error message like on cc-method.js?
```suggestion
                                        fullScreenLoader.stopLoader();
                                        this.messageContainer.addErrorMessage({
                                    message: $t('Something went wrong. Please try again.')
                                });
```